include/aspect/material_model/rheology/drucker_prager.h

```diff
@@ -41,12 +43,12 @@ namespace aspect
         /**
          * List of internal friction angles (phi).
          */
-        std::vector<double> angles_internal_friction;
+        double angles_internal_friction;
```

Comment:
Since you now only return one value, please update the comment (it's no longer a list of angles) and change the name of the parameter to `angle_internal_friction` or `internal_friction_angle`

Reply:
same for cohesions